centerX: 483238.6184840562
centerY: 6674530.097688958
